include/main.h

```diff
@@ -1,64 +1,58 @@
+#ifndef HANGMAN_MAIN_H
+#define HANGMAN_MAIN_H
+
 ///////////////////////////
 // VARIABLE DECLARATIONS //
 ///////////////////////////
 
 // hangman banner
-const char *hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
+char const* const hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
+
+char const* const FRESH = "________\n |/   |\n |\n |\n |\n |\n_|_\n";
+char const* const FIRST = "________\n |/   |\n |   (_)\n |\n |\n |\n_|_\n";
+char const* const SECOND = "________\n |/   |\n |   (_)\n |    |\n |    |\n |\n_|_\n";
+char const* const THIRD = "________\n |/   |\n |   (_)\n |   /|\n |    |\n |\n_|_\n";
+char const* const FOURTH =  "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |\n_|_\n";
+char const* const FIFTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   /\n_|_\n";
+char const* const SIXTH = "________\n |/   |\n |   (_)\n |   /|\\\n |    |\n |   / \\\n_|_\n";
```

Comment:
Using `const` to the left of a pointer variable denotes that the content pointed to is constant. So too does writing `const` to the right of the type in a pointer variable declaration. 
Conventional tradition is to write the `const` on the left, but if you want to declare a pointer variable that both has `const` contents and is also a `const` pointer itself, you now have to read the type declaration in a back-and-forth somewhat circular pattern, leading to confusing patterns as explained [here](https://faculty.cs.niu.edu/~mcmahon/CS241/Notes/reading_declarations.html).
If you simply always write modifiers like `const` to the right of what it is modifying, e.g. immediately to the right of the `char` type and also to the write of the `*` pointer, then reading types becomes a very simple right-to-left order.